lib/adal-angular.js

```diff
@@ -123,9 +123,9 @@
                                         updateDataFromCache(_adal.config.loginResource);
                                         $rootScope.userInfo = _oauthData;
                                         // redirect to login requested page
-                                        var loginStartPage = _adal._getItem(_adal.CONSTANTS.STORAGE.START_PAGE);
+                                        var loginStartPage = _adal._getItem(_adal.CONSTANTS.STORAGE.LOGIN_REQUEST);
                                         if (loginStartPage) {
-                                            $location.url(loginStartPage);
+                                            $window.location = loginStartPage;
```

Comment:
The reason I have to use $window.location is because now loginStartPage is the entire url and not just the part after "#" as it was earlier. $location does not allow to reset the part before "#".